apps/dashboard/pages/[endpoint].tsx

```diff
@@ -236,6 +237,7 @@ const ModelPage: FC<ModelPageProps> = ({ endpoint }) => {
         >
           {endpoint === 'posts' && selectedModel && (
             <Box p={4} rounded="md" bg="white" shadow="md">
+              <TweetGenAI />
```

Comment:
```suggestion
              <TweetGenAI content={(selectedContent as Post).content} />
```

If you want, you can still pass post's content data to use the default content.